x-pack/plugins/transform/public/app/app.tsx

```diff
@@ -60,6 +62,11 @@ export const App: FC<{ history: ScopedHistory }> = ({ history }) => {
 };
 
 export const renderApp = (element: HTMLElement, appDependencies: AppDependencies) => {
+  // TODO Temporary fix to make the Data Grid Histograms in the Transforms Wizard work
+  setDependencyCache({
+    http: appDependencies.http,
+  });
+
```

Comment:
I believe you can avoid this temp fix without significant refactoring.

Reply:
Removed in 3306ff8.